### ISSUE TITLE
DateField calendar popup keyboard navigation does not work with Firefox

### ISSUE THREAD

**Issue by TatuLund** [vaadin/framework#11465]
- Vaadin Framework version, tested with 8.7.0 & 8.62.
- Browser version: Firefox 65.0.1
- Description of the bug

- Minimal reproducible example

Go e.g. Sampler

https://demo.vaadin.com/sampler/#ui/data-input/dates/popup-date-field

Tab multiple times until focus on DateField, 
Press cursor down to open popup
Try to select date with cursor keys

- Expected behavior

Possible to change focused day and select it by enter

- Actual behavior

Focused day cannot be changed

https://demo.vaadin.com/sampler/#ui/data-input/dates/popup-date-field
following information:

**Comment by anasmi** [comment]
The problem was not presented in the previous version of the Firefox (64.0.2)
The same way as DateField doesn't behave correctly anymore, also table doesn't https://demo.vaadin.com/sampler-7.7/#ui/grids-and-trees/table (Clicking down scrolls the Grid, before used to go to another row, which is correct behaviour)
